sql/codegen.go

```diff
@@ -95,15 +112,22 @@ func newFiller(pr *extendedSelect, fts fieldTypes, db *DB) (*filler, error) {
 		}
 		sa := strings.Split(cfg.Addr, ":")
 		r.Host, r.Port, r.Database = sa[0], sa[1], cfg.DBName
-		// FIXME(weiguo): make gohive support Auth
-		r.User, r.Password, r.Auth = cfg.User, cfg.Passwd, "NOSALS"
+		r.User, r.Password = cfg.User, cfg.Passwd
+		r.StandardSelect = removeLastSemicolon(r.StandardSelect)
```

Comment:
Why do we need to remove the last semicolon?

Reply:
In `hive`, a statement with the last semicolon will cause a `ParseException` error.